SPECS/xenopsd.spec

```diff
@@ -95,20 +114,54 @@ cp %{SOURCE6} xenopsd-network-conf
 cp %{SOURCE7} xenopsd-64-conf
 
 %build
+# this is a hack: we build and install two builds into the source
+# directory under root1 and root2. In the install step all we do is
+# copying files from it.
+
+mkdir root1 root2
```

Comment:
Please call these 'normal' and 'cov', or something similar.    root1 and root2 are not very descriptive.